### ISSUE TITLE
Qwen3-VL-Bridge-RT-1 checkpoint issues

### ISSUE THREAD

**Issue by Xiaoxi-Sun** [starVLA/starVLA#324]
Hi, I'd like to raise a couple of issues regarding the Qwen3-VL-Bridge-RT-1 checkpoints:
1. [Qwen3VL-OFT-Bridge-RT-1](https://huggingface.co/StarVLA/Qwen3VL-OFT-Bridge-RT-1) appears to have the wrong version uploaded. The current checkpoint seems to be from 5000 steps, which doesn't look like the intended release version. Could you double-check and re-upload the correct checkpoint?
2. [Qwen3VL-PI_v3-Bridge-RT_1](https://huggingface.co/StarVLA/Qwen3VL-PI_v3-Bridge-RT_1) shows a near-zero success rate on the "put apple in drawer" task in SimplerEnv. Is this expected behavior?

**Comment by Hong-King** [comment]
Same question. I used the official Qwen3-VL-OFT-LIBERO-4in1 model downloaded from the [modelscope](https://www.modelscope.cn/models/starVLA/Qwen3-VL-OFT-LIBERO-4in1), and the success rate in the LIBERO environment was also 0, which is a bit weird.

**Comment by Hong-King** [comment]
https://github.com/user-attachments/assets/d9335816-bbce-4cb9-bef0-c2ebba7d683f

https://github.com/user-attachments/assets/a1f4f2ba-7e64-49fd-a7f0-0f3e02a6af57

**Comment by Hong-King** [comment]
I trained a model based on QWen3.5 (on A100), and test it on LIBERO (run on V100). The result is not so bad, but the task "rollout_open_the_top_drawer_and_put_the_bowl_inside" is always failure.

Framework：QwenOFT
Training Steps：30K
Global Batch Size：4 batch size * 2 GPUS * 16 gradient accumulation steps（ds setting）

https://github.com/user-attachments/assets/9d6ba1cc-9f69-4404-87dd-5b02bf62858d

**Comment by JinhuiYE** [comment]
Thanks for the detailed report! The consistent failure on a single task like rollout_open_the_top_drawer_and_put_the_bowl_inside is a pattern we've seen before and is very likely caused by LIBERO environment version mismatches rather than the model itself.

Key fixes found in the community:

mujoco==3.2.3 — This is the most impactful fix. Newer mujoco versions (e.g. 3.7.0) silently change physics simulation behavior and cause specific tasks to consistently fail. See [#201](https://github.com/starVLA/starVLA/issues/201#issuecomment-4288119939) (4 👍 from community).

numpy==1.24.4 — This is the pinned version in our official LIBERO setup instructions. Mismatched numpy can cause subtle action-space processing issues.

robosuite==1.4.1 — robosuite 1.5+ introduced breaking API changes that affect LIBERO. See [#254](https://github.com/starVLA/starVLA/issues/254).

Please run these inside your LIBERO conda environment:

'''bash
pip install numpy==1.24.4 mujoco==3.2.3 robosuite==1.4.1
'''
This matches the verified setup in our [LIBERO README](https://github.com/starVLA/starVLA/blob/main/examples/LIBERO/README.md#-1-environment-setup). Also, if you haven't already, please confirm your transformers version in the starVLA environment is 4.57.0 — checkpoints are very sensitive to this (ref [#231](https://github.com/starVLA/starVLA/issues/231#issuecomment-4283432010)).

Community experience ([#236](https://github.com/starVLA/starVLA/issues/236#issuecomment-4188221156)) also suggests doing a fresh reinstall of the LIBERO environment if version pinning alone doesn't resolve the issue, since incremental updates over time can leave the environment in an inconsistent state.

**Comment by Hong-King** [comment]
Thank you for your reply.
I checked the version in LIBERO environment, my settings are "numpy==1.24.4 mujoco==3.2.3 robosuite==1.4.0".

BUT the version of "transformers" in the starVLA environment have to be 5.4.0 since I use flash-attention on Qwen3.5, which is incompatible with “transformers==4.57.0”. I tried, version 5.4.0 is the minimum requirement for Qwen3.5. More details please see  https://github.com/Dao-AILab/flash-attention/issues/2381

Furthermore, my LBERO is a completely new and stand-alone environment, and apart from locking the package version according to starVLA's libero-README document, there have been no incremental updates.

**Comment by JinhuiYE** [comment]
Understood. However, please note that transformers==5.4.0 will affect the evaluation of old checkpoints, but it will not affect ckpts that were trained with version 5.4.0.
